spec/models/education_benefits_claim_spec.rb

```diff
@@ -0,0 +1,38 @@
+require "rails_helper"
+
+RSpec.describe EducationBenefitsClaim, type: :model do
+  let(:attributes) do
+    {
+      form: { foo: true }
+    }
+  end
+  subject { described_class.new(attributes) }
+
+  describe "validations" do
+    it { should validate_presence_of(:form) }
```

Comment:
also shoulda matchers have broken quite a few times with new rails versions... particularly with the active record validations. It's just extra cruft, unnecessary... the only thing I've found it particularly useful for is when you want to test dependent destroy -- but i'd sooner write my own rspec matcher for something like that rather than leverage one convenience method from a huge trove of others.